package.json

```diff
@@ -22,14 +22,13 @@
   "author": "pcaversaccio (https://pcaversaccio.com), Matt Solomon (https://mattsolomon.dev)",
   "license": "AGPL-3.0-only",
   "scripts": {
-    "help": "npx hardhat help",
-    "accounts": "npx hardhat accounts",
-    "balances": "npx hardhat balances",
-    "node": "npx hardhat node",
-    "clean": "npx hardhat clean",
+    "clean:hh": "npx hardhat clean",
+    "clean:forge": "forge clean",
     "test": "forge test --out forge-artifacts",
-    "coverage": "npx hardhat coverage",
-    "compile": "npx hardhat compile",
+    "snapshot": "forge snapshot --out forge-artifacts",
+    "coverage": "forge coverage --report summary",
+    "coverage:report": "forge coverage --report lcov && lcov --remove lcov.info 'test/*' --output-file lcov.info --rc lcov_branch_coverage=1 && genhtml lcov.info --branch-coverage --output-dir coverage",
+    "compile:hh": "npx hardhat compile",
     "build": "forge build --out forge-artifacts",
```

Comment:
Similar to above, plus would be nice for these to both be called `compile` or both `build`

Reply:
Implemented via https://github.com/pcaversaccio/createx/pull/18/commits/b8c3646a295418d978ea321ceee83aeb72a907fb.